telegram/ext/filters.py

```diff
@@ -1681,16 +1681,22 @@ class sender_chat(_ChatUserBaseFilter):
         username.
 
         Examples:
-            * To filter for messages forwarded from a channel with ID ``-1234``, use
-              ``MessageHandler(Filters.sender_chat(-1234), callback_method)``.
+            * To filter for messages forwarded to a discussion group from a channel with ID
+              ``-1234``, use ``MessageHandler(Filters.sender_chat(-1234), callback_method)``.
             * To filter for messages of anonymous admins in a super group with username
               ``@anonymous``, use
               ``MessageHandler(Filters.sender_chat(username='anonymous'), callback_method)``.
-            * To filter for messages forwarded from *any* channel, use
+            * To filter for messages forwarded to a discussion group from *any* channel, use
               ``MessageHandler(Filters.sender_chat.channel, callback_method)``.
             * To filter for messages of anonymous admins in *any* super group, use
               ``MessageHandler(Filters.sender_chat.super_group, callback_method)``.
 
+        Note:
+            Remember, sender_chat is also set for messages in a channel as the channel itself,
+            so when your bot is an admin in a channel and the linked discussion group and you
+            filter messages from this channel, you would receive the message twice (once from
```

Comment:
The "and you filter messages from this channel" seems a bit superfluous to me

Reply:
agreed